M2/Macaulay2/packages/Macaulay2Doc.m2

```diff
@@ -12,6 +12,7 @@ newPackage ("Macaulay2Doc",
 	  {Name => "Michael E. Stillman", Email => "mike@math.cornell.edu", HomePage => "http://www.math.cornell.edu/People/Faculty/stillman.html"}
 	  },
      Keywords => {"Miscellaneous"},
+     PackageExports => Core#"pre-installed packages",
```

Comment:
This is necessary in order to get the correct "ways to use" links, but it also slows down installing the examples it seems like:
```m2
 -- making example results for "isModule"                                    -- 4.83991 seconds elapsed
 -- making example results for "leadCoefficient"                             -- 4.83778 seconds elapsed
 -- making example results for "gradedModule"                                -- 4.82856 seconds elapsed
```

@DanGrayson is there a way to force `beginDocumentation()` to end loading the file even if the package isn't installed yet?

Reply:
Or maybe I should add a global variable or command line option for that purpose?

Reply:
I remembered that the main culprit is the Truncation package loading Polyhedra, and given the recent changes with `shared.m2`, I removed Truncation from preloaded packages. Now it should run fast again.

Reply:
> @DanGrayson is there a way to force `beginDocumentation()` to end loading the file even if the package isn't installed yet?

Yes, it could return `end`.  But to what purpose?

Reply:
To make the examples run faster. Currently, when Macaulay2Doc examples are being generated, no other packages are installed, so in particular the "Truncation" package causes a whole bunch of other packages to be read, _including_ their documentation. This PR makes reading the documentation somewhat slower, specifically because `makeDocumentTag` and `(package, Sequence)` are more thorough. These changes are necessary because they fix various issues (see the top comment), so instead removing "Truncations" made the situation is mostly bearable again.